gbpservice/nfp/Automation-Scripts/NFP_SCRIPTS/templates/vpn.template

```diff
@@ -79,9 +79,9 @@
                 },
                 "mtu": 1500,
                 "name": "site_to_site_connection1",
-                "peer_address": "192.168.102.89",
-                "peer_cidrs": ["12.0.0.0/24"],
-                "peer_id": "12.0.0.3",
+                "peer_address": "192.168.102.117",
+                "peer_cidrs": ["19.0.0.0/24"],
+                "peer_id": "192.168.102.117",
```

Comment:
keeping both peer_address and peer_id same does not work in case of heterogeneous vendors, it is good to keep peer_id as the fixed ip of the peer.